templates/competition.html

```diff
@@ -1,15 +1,15 @@
 <template name="competition">
   <div class="jumbotron">
     <div class="container">
-      <h1>{{competition 'competitionName'}}</h1>
+      <h1>{{competitionAttr 'competitionName'}}</h1>
```

Comment:
Just checking that we're on the same page: it would be nice if the data context of this template were the competition object, rather than just an object that has a `competitionId` attribute. Then we could change this `{{competitionAttr 'competitionName'}}` to `{{competitionName}}`. (There are a ton of places where we pass around ids rather than whole objects, so that would be a lot of refactoring to catch them all.)